src/handlers/check_commits/no_mentions.rs

```diff
@@ -0,0 +1,56 @@
+//! Purpose: When opening a PR, or pushing new changes, check for github mentions
+//! in commits and notify the user of our no-mentions in commits policy.
+
+use std::fmt::Write;
+
+use crate::{config::NoMentionsConfig, github::GithubCommit};
+
+pub(super) fn mentions_in_commits(
+    _conf: &NoMentionsConfig,
+    commits: &[GithubCommit],
+) -> Option<String> {
+    let mut mentions_commits = Vec::new();
+
+    for commit in commits {
+        if !parser::get_mentions(&commit.commit.message).is_empty() {
+            mentions_commits.push(&*commit.sha);
+        }
+    }
+
+    if mentions_commits.is_empty() {
+        None
+    } else {
+        Some(mentions_in_commits_warn(mentions_commits))
+    }
+}
+
+fn mentions_in_commits_warn(commits: Vec<&str>) -> String {
+    let mut warning =
+        format!("There are mentions (`@user`) in the following commits, please remove them.\n");
```

Comment:
```suggestion
        format!("There are username mentions (such as `@user`) in the commit messages of the following commits. Please remove the mentions to avoid spamming these users.\n");
```

Reply:
I was trying to avoid a too long line compared to the other warnings, but going to the line between the two sentences makes it okay. Changed.